tests/epyccel/test_builtins.py

```diff
@@ -369,3 +369,71 @@ def f():
     epyc_f = epyccel(f, language=language)
 
     assert epyc_f() == f()
+
+def test_len_list_int(language):
+    def f():
+        a = [1, 2, 3]
+        return len(a)
+
+    epyc_f = epyccel(f, language=language)
+
+    assert epyc_f() == f()
+
+def test_len_list_float(language):
+    def f():
+        a = [1.4, 2.6, 3.5]
+        b = len(a)
```

Comment:
why  use the extra  variable?

Reply:
No particular reason. This just ensures that I test `len` in multiple different settings. There are functions above which use `return len(a)` so this case is also tested.